bitcoin/src/psbt/mod.rs

```diff
@@ -301,7 +298,7 @@ impl Psbt {
         &mut self,
         k: &K,
         secp: &Secp256k1<C>,
-    ) -> Result<SigningKeys, (SigningKeys, SigningErrors)>
+    ) -> Result<SigningKeysMap, (SigningKeysMap, SigningErrors)>
```

Comment:
In 77527e31dcb30c986cca1e2370d4cab83fc32605:

Github won't let me comment directly on it, but the `Returns` section of the doccomment for this method should be rewritten to use backtick links, reduce repetition, to update to the new `SigningKeysMap` type, and to mention that we return internal keys.

Reply:
Oh boy, sloppiest PR this week by far.

Reply:
Lol, well, it was just one doccomment. It just happened to have multiple issues :).